configuration/configuration.go

```diff
@@ -48,6 +48,9 @@ type Configuration struct {
 	// deprecated. Please use Log.Level in the future.
 	Loglevel Loglevel `yaml:"loglevel,omitempty"`
 
+	// Plugins is a path where dynamic plugins are expected to be found
+	Plugins string `yaml:"plugins,omitempty"`
```

Comment:
Well, this might be common:

```
/authplugins/*
/driverplugins/*
```

I would expect that the first name in the plugins path would be taken, just like how normal path-style lookups work.

That said, do we have a way to differentiate the plugin types other than name?